integrations/docker/images/chip-build/Dockerfile

```diff
@@ -178,7 +178,7 @@ RUN set -x \
 
 # Install a known ZAP release
 # Only keep the cli version, since `zap` is 143MB and not usable (UI)
-ENV ZAP_VERSION=v2023.01.06-nightly
+ENV ZAP_VERSION=v2023.01.09-nightly
```

Comment:
What about chip-cert-bins?

Reply:
That needs to get updated in the PR where we start _using_ the new ZAP version.  Which is going to be the one after this one.

Reply:
@tcarmelveilleux https://github.com/project-chip/connectedhomeip/pull/24336 makes that change.